### DISCUSSION TITLE
how can transform image 1 to image 2? please help me to solve the vlnplot problem,

### DISCUSSION THREAD

**Discussion by ggsimidaa168** [satijalab/seurat#6983] [Q&A]
<img width="552" alt="image" src="https://user-images.githubusercontent.com/126305996/221334374-45ff3e8a-dc40-4f4b-924b-4c7d3b003705.png">
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/126305996/221334405-e4736764-2007-4e0a-bf33-f07961697013.png">

**Comment by cgoneill** [comment]
It looks like you're grouping by some sort of cell type annotation, but it's hard to tell without looking at the code you used to generate these plots.

**Reply by ggsimidaa168** [reply]
thanks for your help，i've fixed this out
